wikipedia_cleanup/property_correlation.py

```diff
@@ -259,7 +259,7 @@ def predict_timeframe(
         if len(additional_data) == 0:
             return False
         col_idx = columns.index("value_valid_from")
-        return additional_data[-1:, col_idx] >= first_day_to_predict
+        return additional_data[-1:, col_idx][0] >= first_day_to_predict
```

Comment:
```suggestion
        return additional_data[-1, col_idx] >= first_day_to_predict
```
that should do it and might be even a tiny amount faster